python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -1611,6 +1650,13 @@ def is_multi_gpu(self):
         """
         return True
 
+    def is_remote(self):
```

Comment:
I thought we removed this, or does it need to be here after all? Same question for SG.

Reply:
I did remove this.  It may have come back from a bad merge.  I'll remove it again.  🤦‍♀️